Runtime/Scripts/RGServiceManager.cs

```diff
@@ -240,6 +247,7 @@ public async Task StopBotInstance(long botInstanceId, Action onSuccess, Action o
                 },
                 onFailure: async (f) =>
                 {
+                    RGDebug.LogWarning($"Failed to stop bot instance: {f}");
```

Comment:
All http errors are already logged at debug level, but if we want this specifically logged at warning, we should add the botInstanceId into this message so that we can help them debug from our end.